config.make

```diff
@@ -128,18 +128,38 @@ ZFP_INC = $(ZFP_HOME)/inc
 else
 ZFP_INC = $(ZFP_HOME)/include
 endif
+ifeq ($(wildcard $(ZFP_HOME)/lib),)
+ZFP_LIB = $(ZFP_HOME)/lib64
+else
 ZFP_LIB = $(ZFP_HOME)/lib
+endif
+
+# Check if specified individually the HDF5 include directory,
+# library directory and bin directory separated by commas, i.e. HDF5_HOME=INC,LIB,BIN
 
-HDF5_INC = $(HDF5_HOME)/include
-HDF5_LIB = $(HDF5_HOME)/lib
-HDF5_BIN = $(HDF5_HOME)/bin
+ifneq (,$(findstring ",",$(HDF5_HOME)))
+  HDF5_INC = $(shell echo $(HDF5_HOME) | awk -F'[,]' '{print $$1}')
+  HDF5_LIB = $(shell echo $(HDF5_HOME) | awk -F'[,]' '{print $$2}')
+  HDF5_BIN = $(shell echo $(HDF5_HOME) | awk -F'[,]' '{print $$3}')
+  MAKEVARS =
+else
+
```

Comment:
extra empty line?

Reply:
For example, Ubuntu splits the install into /usr/lib/.../hdf5/serial and the includes in /usr/include/hdf5/serial. The files are not under a single directory which I can set as HDF5_HOME.

Reply:
No worries. I appreciate everything you *did* do here 😄

Reply:
It seems that cmake build can not use an autotools build of zfp. 

Ubuntu available CMake version is too old for zfp, which is why I used make to build zfp for GA. I did not want to build cmake in the CI.

Reply:
Cmake builds do not seem to work with autotools builds of HDF5, it finds HDF5 but does not set lib and include flags for compilation.